seed/analysis_pipelines/eui.py

```diff
@@ -219,15 +220,26 @@ def _run_analysis(self, meter_readings_by_analysis_property_view, analysis_id, p
     progress_data = ProgressData.from_key(progress_data_key)
     progress_data.step('Caclulating EUI')
 
+    Column.objects.get_or_create(
+        is_extra_data=True,
+        column_name="analysis_eui",
+        display_name="Analysis EUI",
+        organization=analysis.organization,
+        table_name='PropertyState',
+    )
+
     for analysis_property_view_id in meter_readings_by_analysis_property_view:
         analysis_property_view = AnalysisPropertyView.objects.get(id=analysis_property_view_id)
         property_state = PropertyState.objects.get(id=analysis_property_view.property_state.id)
```

Comment:
FYI I don't think you'd need to make this query, since `analysis_property_view.property_state` is the same thing as what you're getting